benches/bench_object.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(test)]
```

Comment:
(For the file name) how about bench_pyobject or so?

Reply:
Done 👍